src/index.html

```diff
@@ -14,5 +14,39 @@
   </head>
   <body>
     <h1>Calendar</h1>
+
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
It's not necessary to manually create each day of the calendar in HTML. Consider using a scripting language like JavaScript to dynamically generate these days. This will make your code more maintainable and scalable.